lib/rucio/core/rse.py

```diff
@@ -487,19 +492,22 @@ def get_rses_with_attribute_value(key, value, lookup_key, session=None):
 
 
 @read_session
-def get_rse_attribute(key, rse_id=None, value=None, session=None):
+def get_rse_attribute(key, rse_id=None, value=None, use_cache=True, session=None):
```

Comment:
i don't quite understand why you want to make this optional. shouldn't the cache be used always?

Reply:
I added it because I want to use `get_rse_attributes` in the tests (test_import_export.py) and the cache is also running on travis. I can also replace it there with a normal db query.

Reply:
hmmmm, it's a core function so it should be fine, as long as it's not in the api ;-)